man/systemd.network.xml

```diff
@@ -1322,6 +1322,13 @@
             project='man-pages'><refentrytitle>resolv.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>SetRoutesToDNS=</varname></term>
+          <listitem>
+            <para>When true (the default), the routes to the DNS servers received from the DHCP server
+            will be configured. When <varname>UseDNS=</varname> is disabled, this setting is ignored.</para>
+          </listitem>
+        </varlistentry>
```

Comment:
Hmm, does this need to be configurable? When would one want to set it to false?

Reply:
I have the suspicion that some people might have set their own routes for DNS servers, and we probably should give them a way out from our automatic logic, so I would probably keep this setting

Reply:
Yeah, I'd like to keep this.